modules/37_industry/subsectors/datainput.gms

```diff
@@ -568,4 +550,145 @@ execute_load "input_ref.gdx", vm_demFEsector;
+!! set all others to zero to have rough approximation of power plant output
+p37_specFeDemTarget("fegas","bf","standard")    = sm_eps / (sm_TWa_2_MWh/sm_giga_2_non);    !! Source: DUMMY
+p37_specFeDemTarget("feels","bf","standard")    = sm_eps / (sm_TWa_2_MWh/sm_giga_2_non);    !! Source: DUMMY
+p37_specFeDemTarget("fehos","bf","standard")    = sm_eps / (sm_TWa_2_MWh/sm_giga_2_non);    !! Source: DUMMY
+
+!! per tC for cc tech!!
+p37_specFeDemTarget("feels","bfcc","standard")    = 0.11 * 3.67 / (sm_TWa_2_MWh/sm_giga_2_non);    !! Source: Tsupari2013
+p37_specFeDemTarget("fegas","bfcc","standard")    = 0.92 * 3.67 / (sm_TWa_2_MWh/sm_giga_2_non);    !! Source: Tsupari2013 / Yun2021
+
+!! World Steel Factsheet says no additional equipment needed --> very cheap and no energy demand
+!! IEA Steel Roadmap Fig 2.11 also shows very little additional fuel cost
+p37_specFeDemTarget("feels","idrcc","ng")         = 0.11 * 3.67 / (sm_TWa_2_MWh/sm_giga_2_non);    !! Copy from bfcc
+p37_specFeDemTarget("fegas","idrcc","ng")         = 0.92 * 3.67 / (sm_TWa_2_MWh/sm_giga_2_non);    !! Copy from bfcc
+$endif.cm_subsec_model_steel
+
+*** --------------------------------
+
+p37_mat2ue(all_enty,all_in) = 0.;
+$ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
+p37_mat2ue("sesteel","ue_steel_secondary") = 1.;
+p37_mat2ue("prsteel","ue_steel_primary")   = 1.;
+$endif.cm_subsec_model_steel
+
+*** --------------------------------
+
+p37_captureRate(all_te,opmoPrc) = 0.;
+$ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
+p37_captureRate("bfcc","standard") = 0.73; !! Source: Witecka 2023, Figure 18
+p37_captureRate("idrcc","ng")      = 0.85; !! Source: IEA Steel Roadmap Fig. 2.11
+$endif.cm_subsec_model_steel
+
+*** --------------------------------
+
+p37_priceMat(all_enty) = 0.;
+$ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
+!! IEA STeel Roadmap Fig 1.3 Caption: Scrap price 200-300 $/t
+!! => take 250 $/t, inflation 2005 --> 2020 / 1.33
+p37_priceMat("eafscrap") = 0.188;
+p37_priceMat("bofscrap") = 0.188;
+!! Agora KSV-Rechner: 114 €/tSteel / (1.4 2005$/2023€) / (tn$ /bn t)
+p37_priceMat("ironore")  = 0.081;
+!! Agora KSV-Rechner: 154 €/tSteel / (1.4 2005$/2023€) / (tn$ /bn t)
+p37_priceMat("dripell")  = 0.110;
+$endif.cm_subsec_model_steel
+
+*** --------------------------------
+
+p37_specFeDem(tall,all_regi,all_enty,all_te,opmoPrc) = 0.;
+$ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
+if (cm_startyear eq 2005,
+  vm_outflowPrc.fx('2005',regi,'bof','unheated') = pm_fedemand('2005',regi,'ue_steel_primary');
+  vm_outflowPrc.fx('2005',regi,'bf','standard') = p37_specMatDem("pigiron","bof","unheated") * vm_outflowPrc.l('2005',regi,'bof','unheated');
+  vm_outflowPrc.fx('2005',regi,'eaf','sec') = pm_fedemand('2005',regi,'ue_steel_secondary');
+  vm_outflowPrc.fx('2005',regi,'eaf','pri') = 0.;
+  vm_outflowPrc.fx('2005',regi,'idr','ng') = 0.;
```

Comment:
sounds like abound and not an input?

Reply:
not this one (05_inicap/on/preloop.gms)

Reply:
> not this one (05_inicap/on/preloop.gms).

My bad.

> so I could introduce a parameter (as mentioned above), use it in inicap

Probably best.  Some years down the line some poor PhD will wonder why there is a variable that is not variable at all.

Reply:
I agree, introcuding an additonal parameter is teh cleanest way